sources/dune

```diff
@@ -0,0 +1,12 @@
+(env
+ (dev
+  (flags (:standard
+           -annot -absname -bin-annot
+           -w +A-45-27-4-9-44-32-48-41-6-22
+           )))
+ (release
+  (flags (:standard
+           -annot -absname -bin-annot))
+  (ocamlopt_flags -O3 -unbox-closures))
```

Comment:
Why does the flags "-O3" and "-unbox-closures" are only enable in "release" and not both in "dev" and "release" ?

Reply:
Speed up build time ?

Reply:
Yeah sure. What happen if i want to test the performance of a new optimization in AE on my benchs, do i have to build a release to have the most of the performance offer by my compiler ? or do i use a switch without Flambda enable (since "-O3" has no effect if your switch does not have Flambda.) until i want to bench ?

Reply:
I think we can put `-O3 -unbox-closures` in "dev" as well. Compilation of Alt-Ergo is quite fast, and even faster with dune. As @Coquera saind, these options have probably no effect in Flambda is disabled.

Reply:
done.